cfme/networks/provider/__init__.py

```diff
@@ -28,7 +28,7 @@
 from cfme.utils.log import logger
 
 
-@attr.s
+@attr.s(hash=False)
```

Comment:
**QUESTION**
Is it necessary to change it here?

Reply:
Yes, otherwise python-attrs library forces own `__hash__` function on `NetworkProvider` hence the one from `BaseProvider` (which we actually want)  gets overwritten. I'm quite certain, because it didn't work until I've added the `False` here as well.

Reply:
We have seen this in the past - I think it's OK here